README.md

```diff
@@ -375,6 +375,12 @@ Keeps your at-rules like media queries in the order to defined them.
 
 Ideally, this would be defaulted to `true` and it will be in the next major version. All of the tests expecations need to be updated and probably just drop support for `preserveAtRulesOrder: false`
 
+### `preserveUndefinedVariables` (default: `false`)
```

Comment:
@vithar Can you give an example of why `undefined` isn't desired and is conflicting?

Does the existing `preserve: true` option cover your use case? The `undefined` value will just fallback to the same preserved value your trying to keep around with this new option.

Reply:
![image](https://github.com/MadLittleMods/postcss-css-variables/assets/15692393/6b4cf5fb-0aee-4c21-ae35-f7bfbf7b73ef)
If variable is undefined, the build result is `width: undefined;`. 
maybe that's why